Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -143,10 +141,10 @@ public struct Configuration: Equatable {
                 configPath = fullPath
                 return
             } else {
-                failIfRequired()
+                fail()
```

Comment:
Ah yeah, that could work too, although the point of the local `fail()` function is to keep the failure-related logic in there anyway, so I'll keep it as is.